src/sqlfluff/dialects/dialect_snowflake.py

```diff
@@ -1040,6 +1040,52 @@ class AlterTableTableColumnActionSegment(BaseSegment):
     )
 
 
+@snowflake_dialect.segment()
+class AlterTableClusteringActionSegment(BaseSegment):
+    """ALTER TABLE `clusteringAction` per defined in Snowflake's grammar.
+
+    https://docs.snowflake.com/en/sql-reference/sql/alter-table.html#clustering-actions-clusteringaction
+
+    If possible, please match the order of this sequence with what's defined in Snowflake's
```

Comment:
The wording of this sentence seems strange. Why "please match"?

Reply:
I just copied the phrase from the similar `AlterTableTableColumnActionSegment` introduced [here](https://github.com/sqlfluff/sqlfluff/pull/2267#discussion_r781325367) 🤷

Reply:
tbh I think it's redundant anyway so just gonna remove it 👍